subsys/bluetooth/host/gatt.c

```diff
@@ -4989,6 +5043,18 @@ void bt_gatt_connected(struct bt_conn *conn)
 
 #if defined(CONFIG_BT_GATT_CLIENT)
 	add_subscriptions(conn);
+
+#if defined(CONFIG_BT_SETTINGS) && defined(CONFIG_BT_SMP)
+	static struct bt_conn_cb gatt_conn_cb = {
+		.identity_resolved	= bt_gatt_identity_resolved
```

Comment:
minor: just use a space before `=` (now you have a tab, and tabs should only ever exist at the very beginning of a line)

Reply:
Also, add a comma at the end (I know, not necessary, but it makes diffs easier to read if/when new callbacks get added here)

Reply:
Done.